service/manager.go

```diff
@@ -83,7 +83,7 @@ func newManager(builder *Builder) (Manager, error) {
 		// If the user didn't pass in a configuration provider, load the standard.
 		// Bypassing standard config load is pretty much only used for tests, although it could be
 		// useful in certain circumstances.
-		m.configProvider = config.Load()
+		m.configProvider = config.DefaultLoader.Load()
```

Comment:
we should preserve the `Load()` package level function which just calls DefaultLoader, if this is one of the "breaking" changes to the API